### PR TITLE
completed javascript-light-bulb challenge.

### PR DESCRIPTION
**Pull request by vkahlon**
Screen Shot 2021-12-14 at 2 50 34 PM

### CODE REVIEW

File: javascript-light-bulb/main.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
+var clickCount = 0;
+function letThereBeLight(event) {
+  clickCount++;
+  if (clickCount % 2 === 1) {
```
Comment (codingcodymiller) [inline javascript-light-bulb/main.js:4]: In your code, you are using the count here only to determine if it is even or odd. There's another data type that is better to use for values that are one thing or the other thing: a boolean. You should instead make a boolean variable to keep track of if the light is currently on or not.

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

**Comment by vkahlon** [comment]
I converted the primary if statement to boolean, I had to convert the number system into boolean system, but the benefit is that I can keep count of the clicks, if I have to.

**Comment by vkahlon** [comment]
Do not grade this latest commit!

**Comment by vkahlon** [comment]
Now it's ready to be graded.

**Review by codingcodymiller — APPROVED** [review]

**Pull request merged**